common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4018,6 +4022,32 @@ object CelebornConf extends Logging {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("64m")
 
+  val CLIENT_PUSH_SORT_USE_ADAPTIVE_MEMORY_THRESHOLD: ConfigEntry[Boolean] =
+    buildConf("celeborn.client.spark.push.sort.memory.useAdaptiveThreshold")
+      .withAlternative("celeborn.push.sortMemory.useAdaptiveThreshold")
+      .categories("client")
+      .doc("adaptively adjust threshold for sort shuffle writer's memory threshold")
+      .version("0.5.0")
+      .booleanConf
+      .createWithDefault(false)
+
+  val CLIENT_PUSH_SORT_SMALL_PUSH_TOLERATE_FACTOR: ConfigEntry[Double] =
+    buildConf("celeborn.client.spark.push.sort.smallPushTolerateFactor")
+      .withAlternative("celeborn.push.sortMemory.adaptiveThreshold")
+      .categories("client")
+      .doc("only be in effect when celeborn.client.spark.push.sort.memory.adaptiveThreshold is" +
```

Comment:
From a user perspective, based on this configuration description, the user has no way of knowing how to configure this factor.

Reply:
updated